### PR TITLE
Update BMLacq821

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
I updated the record for BMLacq821. To my surprise, the transcription by transcribus was inserted in the ms-file. It may be necessary to split this record into two to have the transcribus record seperately.

### CODE REVIEW

**Comment by CarstenHoffmannMarburg** [comment]
I wanted to encode Josef in ms_i6, but I did not find an ID for him. It is supposed to be Joseph, the son of Jacob and Rachel, who is registered as Q145746 in Wikiquote. Is his record missing in Bm?

I checked EMIP00227 and LIT6751PPAynat and discovered some similarities to ms_i6 in the manuscript and in the record respectively. However, I saw, that the link [www.betamasaheft.eu/LIT6751PPAynat](https://betamasaheft.eu/works/LIT6751PPAynat/main) is not available at the moment.

**Comment by thea-m** [comment]
It's PRS5763Joseph. I've added the wikidata ID and the Bm ID to wikidata.

File: FlorenceBML/BMLacq821/BMLacq821.xml [hunk 1]
```diff
@@ -33,8 +33,207 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                				<condition key="good"/>
+                			</supportDesc>
+                			<layoutDesc><layout columns="1" writtenLines="182">
+                				<ab type="pricking">No pricking/></ab>
+                				<ab type="ruling">No ruling</ab>
+                			</layout>
+                			</layoutDesc>
+                		</objectDesc>
+                		<handDesc><handNote xml:id="h1" script="Ethiopic">
+                			<ab type="script"/>
```
Comment (eu-genia) [inline FlorenceBML/BMLacq821/BMLacq821.xml:187]: why this `ab`?

**Review by eu-genia — CHANGES_REQUESTED** [review]

**Comment by eu-genia** [comment]
No, it is not necessary to split the record at all.
We split only when the transcriptions are too long for github.
If you see an error on calling an entry (as in LIT6751PPAynat) it is not because the "link is unavailable" but because there is an encoding issue or a problem with the code, and an issue is needed to settle the bug (done).

File: FlorenceBML/BMLacq821/BMLacq821.xml (hunk 1 above)
Comment (CarstenHoffmannMarburg) [reply FlorenceBML/BMLacq821/BMLacq821.xml:187]: Deleted `<ab type="script"/>`

**Review by eu-genia — APPROVED** [review]

**Pull request merged**